### ISSUE TITLE
The object is not broken by hand in mineclone 2

### ISSUE THREAD

**Issue by wuniversales** [minetest-mods/pontoons#8]
Object is not broken in creative mode by hand in mineclone 2.
In survival mode, it is impossible to break it too.

**Comment by minertestdude** [comment]
Thanks, I found and fixed it. Mcl2 seems to ignore groups and uses { _mcl_blast_resistance = ..,   _mcl_hardness = .. } to determine how easy things break. See games/mineclone2/mods/ITEMS/mcl_core/nodes_base.lua

**Comment by minertestdude** [comment]
No, all wrong, now I know what is going on. If group gets "level=2" for some reason it makes block indestructable. Any other group is either information only, or allows/prevents drops when you dig it with specific tool. Anything can be dug, the speed depends upon _mcl_hardness only.
